includes/fs-essential-functions.php

```diff
@@ -36,6 +36,39 @@ function fs_normalize_path( $path ) {
 		}
 	}
 
+	if ( ! function_exists( 'fs_get_plugins' ) ) {
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.2.1
+         *
+         * @param string $plugin_folder
+         * @param bool   $delete_cache
+         *
+         * @return array
+         */
+	    function fs_get_plugins( $plugin_folder = '', $delete_cache = false ) {

```

Comment:
@fajardoleo since all the calls to `fs_get_plugins()` always use an empty string for the 1st param, there's no need to have it at all. If we'll need to use it in the future, we'll add it.